modules/dhcp_common/pingable.rb

```diff
@@ -0,0 +1,58 @@
+require 'timeout'
+require 'socket'
+
+module Proxy::DHCP::Pingable
```

Comment:
tcp and icmp ping code has been extracted into a dedicated module, but the original code is still there: https://github.com/theforeman/smart-proxy/blob/develop/modules/dhcp_common/free_ips.rb#L103. Could you remove it and use the module instead?

Reply:
Sorry I missed that code move after my last rebase. I've removed the code from free_ips and included the module instead.